src/App.js

```diff
@@ -1,82 +1,186 @@
-                    </td>
-                  </tr>
-                  <tr>
-                    <td>Giơi Tính:</td>
-                    <td>
-                      <input type="radio" name="gender" value="nam"/>Nam <br/>
-                      <input type="radio" name="gender" value="nu"/>Nữ
-                    </td>
-                  </tr>
-                  <tr>
-                    <td>Sơ thích:</td>
-                    <td>
-                      <input type="checkbox" name="hobby" id=""/>Đá bóng <br/>
-                      <input type="checkbox" name="hobby" id=""/>Cầu lông <br/>
-                      <input type="checkbox" name="hobby" id="" checked/>Đua thuyền <br/>
-                    </td>
-                  </tr>
-                  <tr>
-                    <td>Ghi chú:</td>
                     <td>
-                      <textarea name="ghichu" id="ghichu" cols="20" rows="3"></textarea>
+                      <input type="text" name="name"
+                        value={this.state.name}
+                        onChange={this.handldeChange}
+                        placeholder="Họ và Tên"/>
                     </td>
-                  </tr>
-                </tbody>
-              </table>
-              <input type="submit" name="dangky" id="dangky" value="Đăng ký"></input>
-            </div>
+                </tr>
+                <tr>
+                  <td>Email:</td>
+                  <td>
+                    <input type="email" name="email"
+                      value={this.state.email}
+                      onChange={this.handldeChange}
+                      placeholder="Nhập Email"/>
+                  </td>
+                </tr>
+                <tr>
+                  <td>Mật khẩu:</td>
+                  <td>
+                    <input type="password" name="password"
+                      value={this.state.password}
+                      onChange={this.handldeChange}
+                      placeholder="Nhập Password"/>
+                  </td>
+                </tr>
+                <tr>
+                  <td>Avatar:</td>
+                  <td>
+                    <input type="file" name="file"
+                      value={this.state.file}
+                      onChange={this.handldeChange}
+                      multiple="multiple"/>
+                  </td>
+                </tr>
+                <tr>
+                  <td>Số điện thoại:</td>
+                  <td>
+                      <input type="number" name="number"
+                        value={this.state.number}
+                        onChange={this.handldeChange}
+                        placeholder="Nhập số điện thoại"/>
+                  </td>
+                </tr>
+                <tr>
+                  <td>Tuổi:</td>
+                  <td>
+                      <select name="age" value={this.state.age} onChange={this.handldeChange}>
+                          <option value="age:1">1 tuổi</option>
```

Comment:
Value trong trường hợp này chỉ nên là 1 số (1 number) thôi
```suggestion
                          <option value="1">1 tuổi</option>
```